disruptor.go

```diff
@@ -40,18 +37,11 @@ var (
 
 // NewModuleInstance returns a new instance of the disruptor module for each VU.
 func (*RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
-	kubeConfigPath, err := getKubernetesConfigPath()
-	if err != nil {
-		common.Throw(vu.Runtime(), fmt.Errorf("error getting kubernetes config path: %w", err))
-	}
-	cfg := kubernetes.Config{
```

Comment:
I think this change is introducing some unnecessary complexity in the `kubernetes` package.  As the necessity of having the `newWithContext` function.  My suggestion is to still create the `Config` struct and pass it to the `new` function and inside that function handle the case where the `kubeconfig` parameter is not provided.

Reply:
What do you think? Should I do something different?

Reply:
Hi @mcandeia Sorry for the back and forth. I think the `Config` structure is not really needed. I made some refactoring [here](https://github.com/pablochacin/xk6-disruptor/commit/6225beee087fa7ab3aa504ca4cf464de9ff7e817) and now I think the code is cleaner and the different ways of creating a kubernetes.Kubernetes are clearly separated.

Feel free to pick the changes from there, or if you don't mind, I can try to push that commit to your PR.